style.css

```diff
@@ -764,3 +754,302 @@ button:hover {
 .link-contact:hover {
   color: var(--color-primary);
 }
+
+/* Work PopUps */
+
+.card-work-detail {
+  position: fixed;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-left: 10%;
+  height: 1000px;
+  margin-top: 11px;
+  top: 1px;
+  background: #fff;
+  border: 1px solid #ebecf0;
+  box-shadow: 1px 48px 48px rgba(37, 47, 137, 0.08);
+  border-radius: 8px;
+}
```

Comment:
Regarding the implementation of pop-ups on mobile screens:
 - the layout, for now, is completely different;
 - unfortunately, even with this layout, I don't have access to all pop-up features on mobile screens;

Your implementation | Figma Design
:------:|:------:
![image](https://user-images.githubusercontent.com/83596135/167907642-9d9d4ad8-4328-4833-b17b-d12f90a0a46c.png) | ![image](https://user-images.githubusercontent.com/83596135/167907696-33165345-d6be-48aa-bd89-4bf29e34af9d.png)

- [ ] Kindly, refactor the design of pop-up for mobile to match the design from Figma as close as possible;

Reply:
- [ ] Hey, as required by the previous reviewer, Please, refactor your styling to make your card at the center of the screen. 
You got to fix this by playing with the width property you added.
- Your modal is having a horizontal scroll effect, kindly fix this to improve the user experience.
- Your design doesn't match the Figma design yet. please fix this as required by the previous reviewer